go.mod

```diff
@@ -0,0 +1,19 @@
+module ao
+
+go 1.14
+
+require (
```

Comment:
Er versjonen på avhengighetene ok.
Burde vi lage en sak på å oppgradere avhengigheter?

Reply:
Jeg tenker at det uansett bør være periodiske jobber med å oppdatere versjoner på avhengigheter.  Leveransen har uansett en kompleksitet som gjær at jeg ikke ville gjøre for mange ulike typer endringer på en gang. Av samme grunn valgte jeg å ikke gjøre store endringer i Makefile. Dersom feil oppstår, så har man en enklere jobb med å finne feilen dersom det er en relativt klar avgrensning av hva som er endret i leveransen.